### ISSUE TITLE
Subscribing to Meetup.com calendar links doesn't work

### ISSUE THREAD

**Issue by Bugsbane** [nextcloud/calendar#78]
I'm not sure if this is a bug or a feature request, but basically Meetup.com gives users live updated links to all events in their meetup groups, in 4 different formats ("Google Calendar","Outlook Exchange","iCal" and "RSS"). I'm not sure if any of them are technically webcal or not. If they are, then there seems to be a bug, as none of them add events in Nextcloud calendar. If none of these *are* valid webcal links, them just let me know and I'll file a new issue requesting Google Calendar or iCal support.

I'm not somewhere I can access the server by click right now, so I haven't filled in all text specs. I can later if needed.

There are two errors in the logs, but neither looks calendar related (one is in sabredav and the other a php 404 error about a specific image file)

### Steps to reproduce
1. Get calendar URL from meetup.com calendar > export events page
2. Add to Nextcloud calendar via New Subscription button
3. Reload page

### Expected behaviour
Tell us what should happen
One of the calendar link options should add the Meetup.com events to Nextcloud Calendar

### Actual behaviour
The first two options (Google Calendar link and Outlook Exchange link) show up in Nextcloud Calendar with the correct name ("My Events") but no events ever show in the actual calendar view. The other two (iCal and RSS) both return errors. Something about lacking http authentication for iCal and not being a valid webcal URL for the RSS link.

### Server configuration
**Operating system**:
Debian 8

**Web server:**
Nginx

**Database:**
Mysql

**PHP version:**
7

**Server version:**
10

**Calendar version:**
1.4

**Updated from an older installed version or fresh install:**
Updated. Originally I started with owncloud 8, way back, and have applied updates since, migrating from ownCloud 8.2.3 to Nextcloud 9.0.52 and upgrading to Nextcloud 10 as updates were released.

**Signing status (ownCloud/Nextcloud 9.0 and above):**
Passes signing checks

```
Login as admin user into your cloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**
The default Nextcloud apps and:
Bookmarks
Mail template editor
Video player
Bookmarks
Notes
News
Documents
```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your instance's installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your instance's installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** yes/no
No

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
LDAP

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your instance's installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

**CalDAV-clients:**

#### Web server error log
```
Insert your webserver log here
```

#### Log file (data/owncloud.log)
```
Insert your ownCloud.log file here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by tcitworld** [comment]
Adding an iCal link should work. If you get a 
> Something about lacking http authentication for iCal

then it's related to https://github.com/nextcloud/calendar/issues/70

**Comment by georgehrke** [comment]
Yes, please try the ical link as @tcitworld suggested.
Generally speaking I don't think this is an issue for the bugtracker, rather it's a perfect match for http://help.nextcloud.com

**Comment by Bugsbane** [comment]
Hmmm... Actually, now I retry with the iCal link (sans username and password) it isn't giving me any error messages at all. It just doesn't show any events. Furthermore, I notice that if I copy the link in my browser and then log out of Meetup.com and then go to that url, it still sends me a vcal file.

I tried adding my username and passowrd at the start of the url, ie https://myusername:mypassword@www.meetup.com/etc/etc but it didn't change anything. Looking at the logs the webdav error I saw before is actually linked to another user (who uses the desktop sync client), not my user.
